src/java/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -318,11 +323,20 @@ else if(doRefMetrics) {
 
                                 // With both reads mapped we can see if this pair is chimeric
                                 if (Math.abs(record.getInferredInsertSize()) > maxInsertSize ||
-                                        !record.getReferenceIndex().equals(record.getMateReferenceIndex())) {
+                                        !record.getReferenceIndex().equals(record.getMateReferenceIndex()) ||
+                                        SamPairUtil.getPairOrientation(record) != expectedOrientation ||
```

Comment:
@eitanbanks - yes, you are correct about SVs.  The problem is that, similar to trying to calculate base error rate, without going through a highly accurate variant calling process, it's impossible to tell apart read pairs that appear to come from different parts of the genome due to a biological structural variation, and read pairs that are the product of erroneous lab processes joining two or more disparate molecules together.

The same issue also existed with the prior definition.  As I see it @nh13's change takes us from detecting only anomalous inserts where the break is in between the two reads, to all anomalous inserts.